tests/Microsoft.Fx.Portability.Tests/ApiPortServiceTests.cs

```diff
@@ -15,5 +24,45 @@ public void VerifyParameterChecks()
             Assert.Throws<ArgumentOutOfRangeException>(() => new ApiPortService(string.Empty, new ProductInformation("")));
             Assert.Throws<ArgumentOutOfRangeException>(() => new ApiPortService(" \t", new ProductInformation("")));
         }
+
+        [Fact(Skip = SkipReason)]
+        public void ApiPortService_GetDocIdsWithValidDocId()
+        {
+            var apiPortService = new ApiPortService(
```

Comment:
These tests should be rewritten with an HttpClient mock - create an overload of ApiPortService that takes an HttpMessageHandler and then you can mock it fairly easily.